pkg/controller/dataimportcron-controller.go

```diff
@@ -302,22 +322,32 @@ func (r *DataImportCronReconciler) updateDataSourceOnSuccess(ctx context.Context
 			return err
 		}
 	}
-	if !isOwner(dataSource, dataImportCron) {
-		dataSource.OwnerReferences = append(dataSource.OwnerReferences, metav1.OwnerReference{
-			APIVersion: dataImportCron.APIVersion,
-			Kind:       dataImportCron.Kind,
-			Name:       dataImportCron.Name,
-			UID:        dataImportCron.UID,
-		})
-	}
-	sourcePVC := &cdiv1.DataVolumeSourcePVC{
-		Namespace: dataImportCron.Namespace,
-		Name:      dataImportCron.Status.CurrentImports[0].DataVolumeName,
+	dataSourceCopy := dataSource.DeepCopy()
+
+	if dataSource.Labels == nil {
```

Comment:
@awels For all the CRs created here (CronJob, Job, DataSouce, DVs...)? Also add to new DataImportCron? I don't see you add it to created DVs in its controller.

Reply:
Yes, we have https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/controller/config-controller.go#L118
It would make sense to label DataSources, DataImportCrons, CronJob/Jobs, DVs (that are created for corn purpose)

Reply:
Added